src/Node.cpp

```diff
@@ -91,6 +91,17 @@ void Node::processRxQueue()
 
     if(result == 1)
     {
+      /* Check whether or not the incoming transfer is a message transfer,
+       * if the incoming port id has been subscribed too and, if all those
+       * preconditions hold true, invoke the required transfer received
+       * callback.
+       */
+      if (transfer.metadata.transfer_kind == CanardTransferKindMessage)
+      {
+        if (_msg_subscription_map.count(transfer.metadata.port_id) > 0)
+          _msg_subscription_map.at(transfer.metadata.port_id)->onTransferReceived(transfer);
```

Comment:
I think this should be replaced with [`find()`](https://en.cppreference.com/w/cpp/container/map/find) to avoid double lookup.

Reply:
Done in 03a2099a84c8c740de883467fdc58b5dee594d1f :heavy_check_mark: